src/sessionization.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import division
+import sys
+import traceback
+import datetime
+
+# February 12, 2019, Helena Zhang
+# written and tested in Python 2.7.10
+
+# This script calculates the session
+# Inputs:
+# 1. log.csv, a CSV log file
+# 2. inactivity_period.txt
+# Check README.md for more information.
+
+############################################
+# EDITABLE PARAMETERS
+#
+# to get verbose messages, set DEBUG to True
+DEBUG=False
```

Comment:
I like having debug mode! Another way to do it is to log everything using the `logging` library, then have a command line argument that controls the logging level.